src/ethereum_test_tools/rpc/rpc.py

```diff
@@ -8,6 +8,7 @@
 import requests
 from tenacity import retry, stop_after_attempt, wait_exponential
 
+from cli.gentest import RequestManager
```

Comment:
This introduces a circular dependency, I think we should avoid using a `RequestManager` object here in rpc.py. 

The easiest solution would be to build a dict containing the required parameters and pass this dictionary to `debug_trace_call` as an argument.

We could consider defining a small pydantic BaseModel for these parameters, but it could be overkill in this case - a dict should be fine for now. But something we could consider adding later on.

Reply:
Ok, I'll try to tackle it